tests/data/pluto_isentropic_vortex/meta00.yaml

```diff
@@ -0,0 +1,16 @@
+id: orszag_tang
+attrs:
+  path: data.0010.vtk
+  has_geometry: true
+  geometry: cartesian
+  dimensionality: 2
+  kind: plutoVTK
```

Comment:
the fact that we need to distinguish two varieties of pluto files indicate that we should probably have more attributes (`code` and `format` seem like reasonable names)

Reply:
handled in #221